### ISSUE TITLE
ReferenceError: TextEncoder is not defined

### ISSUE THREAD

**Issue by alyatis85** [Ylianst/MeshCentral#4500]
**Error in the servererror log since applying patch 1.0.81**

```
ReferenceError: TextEncoder is not defined
/node_modules/whatwg-url/lib/encoding.js:2
const utf8Encoder = new TextEncoder();

ReferenceError: TextEncoder is not defined
    at Object.<anonymous> (/home/alyatis/node_modules/whatwg-url/lib/encoding.js:2:21)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/alyatis/node_modules/whatwg-url/lib/url-state-machine.js:5:34)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
```

 - OS: Ubuntu
 - Virtualization: Google
 - Version: 1.0.81

**Comment by Ylianst** [comment]
Duplicate of #4496 4499

**Comment by Ylianst** [comment]
Just published a fix for #4499.

**Comment by alyatis85** [comment]
**I confirmed it is fixed.** 
Thank you very much Ylian.
